spec/puppet_version_spec.rb

```diff
@@ -15,8 +15,8 @@
         SemanticPuppet::VersionRange.parse(version_requirement)
       end
 
-      SUPPORTED_PUPPET_VERSIONS.each do |puppet_version|
-        it "supports Puppet #{puppet_version}" do
+      VERSIONS.each do |puppet_version|
+        it "should support Puppet #{puppet_version}" do
```

Comment:
Isn't this quite the other way around of what RuboCop suggests? It literally reverts 31782b64031ad73afa5a0349d52891a633587594, which was flagged by rubocop-rspec.